### PR TITLE
Suggested changes to Describing info

### PR DESCRIPTION
**Pull request by swapnilogale**

### CODE REVIEW

**Comment by swapnilogale** [comment]
@pvgenuchten can you review this when you get a chance too? Or maybe Victor can help?

File: source/user-guide/describing-information/creating-metadata.rst [hunk 1]
```diff
@@ -3,31 +3,27 @@
...
-
+1. From the home page, go to the *contribute* section or directly click on ``add new record`` menu.
 
 .. figure:: img/add-new-record.png
 
-
-Then, start with a metadata
-template (see :ref:`creating-templates`), choose a group and click *create*.
-
+2. From the metadata template list, select a template (see :ref:`creating-templates`), choose a group from the In dropdiwn and click *Create*.
```
Comment (volaya) [inline source/user-guide/describing-information/creating-metadata.rst]: *In* dropdown

File: source/user-guide/describing-information/deleting-metadata.rst [hunk 2]
```diff
@@ -13,8 +13,5 @@ There are different ways to remove a record:
 - In the ``contribute`` section click the ``delete`` button for a record.
```
Comment (volaya) [inline source/user-guide/describing-information/deleting-metadata.rst]: looks like highlighting of elements is not consistent. Sometimes double quotes are used, and others you use asterisks. Maybe better to use :guilabel:`[element]` to have a more consistent approach and better semantics in the text

**Comment by volaya** [comment]
I added a couple of minor notes, but the changes look good to me

**Pull request merged**